proposals/solid-oidc/index.bs

```diff
@@ -181,9 +181,16 @@ SHOULD use a WebID as their Client ID.
 
 ## WebID ## {#clientids-webid}
 
-The WebID itself MUST resolve to an RDF document. That document SHOULD be available as JSON-LD with an `@context`
-that includes `https://www.w3.org/ns/solid/oidc-context.jsonld` but which otherwise produces a JSON serialization of an OIDC
-client registration, per the definition of client registration metadata from [[!RFC7591]] section 2.
+A client's WebID resource MUST be serialized as an `application/ld+json` document when
+the request includes an Accept header, unless content negotiation requires a different
+outcome.
```

Comment:
I may be nitpicking but I find "includes it in the `Accept` header" (or "includes the media type in the `Accept` header" to more precise than "includes an `Accept` header" since it isn't the mere existence of the `Accept` header (irrespective of the field-value) that results in generating a JSON-LD representation but that because `application/ld+json` is in the field-value.

Reply:
Do we even need to mention `Accept` header or we could just keep that part about content negotiation?

Reply:
The mention of the `Accept` header has been removed